tools/underwater-gps.py

```diff
@@ -194,18 +203,24 @@ def notifyPutResponse(response, *args, **kwargs):
         url = gpsUrl + "/api/v1/position/global"
         print('requesting data from', url)
         request = grequests.get(url, session=s, hooks={'response': processLocatorPosition})
-        job = grequests.send(request)
+        if request:
```

Comment:
Can this ever be None?
https://github.com/spyoungtech/grequests/blob/master/grequests.py#L8

It looks like it always returns a AsyncRequest instance. Networking errors will show in the callback

Reply:
Removed.